CMakeLists.txt

```
@@ -340,10 +340,15 @@ elseif(CELERITAS_USE_HIP)
       OFF
     )
   endif()
-  find_library(ROCTX_LIBRARY roctx64)
+  # Include the default ROCm installation path in the search
+  find_library(ROCTX_LIBRARY roctx64 PATHS /opt/rocm/lib)
   if(ROCTX_LIBRARY)
     set(CELERITAS_HAVE_ROCTX ON)
   endif()
+  find_file(HIPCUB hipcub_version.hpp PATHS /opt/rocm/include/hipcub)
+  if(NOT HIPCUB STREQUAL "HIPCUB-NOTFOUND")
```

I think this should also work:
```suggestion
  if(HIPCUB)
```

Also this and the next two lines (and this change could be made for the ROCTX flag too!) could be:
```cmake
celeritas_set_onoff(CELERITAS_HAVE_HIPCUB ${HIPCUB})
```